css-view-transitions-1/Overview.bs

```diff
@@ -1101,19 +1070,81 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 				except the [=view-transition pseudo-elements=],
 				are not painted (as if they had visibility: hidden)
 				and do not respond to hit-testing (as if they had pointer-events: none) until |new| exists.
+
+			1. If neither of |capturedElement|'s [=captured element/old image=] or [=captured element/new element=] is null:
+
+				1. Let |transform| be |capturedElement|'s [=old styles=]'s 'transform' property.
```

Comment:
Where is old styles defined? I know this has moved from elsewhere, but is this correct? We're not using the captured element's style and read its transform property, but rather find the transform it has in screen space and use that as the transform. Similarly for width/height, it could be a flex grow with no explicit width, so what does style width mean here?

Reply:
I've replaced `[=old styles=]` with `[=captured element/old styles=]`. It's set in "setup view transition".

I think the confusion here is it feels like a direct reference to the old styles. Would it be better if the prose was:

> ```
> 1. Let |transform| be |capturedElement|'s [=captured element/old offset transform=].
> ```